client/src/components/RelatedProducts/Cards/Features.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+
+// adds all the features in to one array
+// then compares if item has feature or not
+function Features({ item1, item2 }) {
+  const FeatArray = item1.features.concat(item2.features);
+
+  const item1Array = [];
+  item1.features.map((item) => item1Array.push(item));
+
+  const item2Array = [];
+  item2.features.map((item) => item2Array.push(item));
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>{item1.name}</th>
+          <th> </th>
+          <th>{item2.name}</th>
+
+        </tr>
+      </thead>
+      <tbody>
+        {FeatArray.map((item) => (
+          <tr>
```

Comment:
The `tr` here needs a `key` property in order to not throw an error in the console.

Reply:
Fixed and added the key property